python/demos/UserContributions/README.md

```diff
@@ -0,0 +1,19 @@
+# User Contributions
+A sample app that uses MediaWiki Action [API:Usercontribs](https://www.mediawiki.org/wiki/API:Usercontribs) allows you to see the latest top 50 edits made by a user. This app uses Flask Framework as backend and Bootstrap as frontend.
```

Comment:
May be you don't need to mention Bootstrap here, as you are not really using the framework.